unable to build on FreeBSD 10.1R amd64

During linking, fails with errors `undefined reference to 'WIFEXITED'` after implicit function declaration warning:

```
/home/dch/src/www/h2o/src/main.c:235:11: warning: implicit declaration of function 'WIFEXITED' is invalid in C99 [-Wimplicit-function-declaration]
    if (!(WIFEXITED(child_status) && WEXITSTATUS(child_status) == 0))
```

Full log:

```
dch 😈  ~/s/w/h2o git:master ❯❯❯git clean -fdx
dch 😈  ~/s/w/h2o git:master ❯❯❯gsh | head -1
commit e062a70e402cffdd3aebf8526e11f0c62f384e7b
dch 😈  ~/s/w/h2o git:master ❯❯❯cmake -DCMAKE_INSTALL_PREFIX=/usr/local .
-- The C compiler identification is Clang 3.4.1
-- The CXX compiler identification is Clang 3.4.1
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/CC
-- Check for working CXX compiler: /usr/bin/CC -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for include file pthread.h
-- Looking for include file pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Found OpenSSL: /usr/lib/libssl.so;/usr/lib/libcrypto.so (found version "1.0.1j")
-- Found LIBUV: /usr/local/lib/libuv.so
-- Found LIBYAML: /usr/local/lib/libyaml.so
-- Could NOT find WSLAY (missing:  WSLAY_LIBRARIES WSLAY_INCLUDE_DIR)
CMake Warning (dev) at CMakeLists.txt:159 (ADD_CUSTOM_TARGET):
  Policy CMP0037 is not set: Target names should not be reserved and should
  match a validity pattern.  Run "cmake --help-policy CMP0037" for policy
  details.  Use the cmake_policy command to set the policy and suppress this
  warning.

  The target name "test" is reserved or not valid for certain CMake features,
  such as generator expressions, and may result in undefined behavior.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Configuring done
-- Generating done
-- Build files have been written to: /home/dch/src/www/h2o
dch 😈  ~/s/w/h2o git:master ❯❯❯make
Scanning dependencies of target h2o
[  2%] Building C object CMakeFiles/h2o.dir/deps/picohttpparser/picohttpparser.c.o
[  4%] Building C object CMakeFiles/h2o.dir/lib/common/memory.c.o
[  4%] Building C object CMakeFiles/h2o.dir/lib/common/http1client.c.o
[  6%] Building C object CMakeFiles/h2o.dir/lib/common/serverutil.c.o
[  8%] Building C object CMakeFiles/h2o.dir/lib/common/socket.c.o
[ 10%] Building C object CMakeFiles/h2o.dir/lib/common/socketpool.c.o
[ 10%] Building C object CMakeFiles/h2o.dir/lib/common/string.c.o
[ 12%] Building C object CMakeFiles/h2o.dir/lib/common/timeout.c.o
[ 14%] Building C object CMakeFiles/h2o.dir/lib/core/config.c.o
[ 17%] Building C object CMakeFiles/h2o.dir/lib/core/configurator.c.o
[ 17%] Building C object CMakeFiles/h2o.dir/lib/core/context.c.o
[ 19%] Building C object CMakeFiles/h2o.dir/lib/core/headers.c.o
[ 21%] Building C object CMakeFiles/h2o.dir/lib/core/request.c.o
[ 23%] Building C object CMakeFiles/h2o.dir/lib/core/token.c.o
[ 23%] Building C object CMakeFiles/h2o.dir/lib/core/util.c.o
[ 25%] Building C object CMakeFiles/h2o.dir/lib/handler/access_log.c.o
[ 27%] Building C object CMakeFiles/h2o.dir/lib/handler/chunked.c.o
[ 29%] Building C object CMakeFiles/h2o.dir/lib/handler/expires.c.o
[ 29%] Building C object CMakeFiles/h2o.dir/lib/handler/file.c.o
[ 31%] Building C object CMakeFiles/h2o.dir/lib/handler/mimemap.c.o
[ 34%] Building C object CMakeFiles/h2o.dir/lib/handler/proxy.c.o
[ 36%] Building C object CMakeFiles/h2o.dir/lib/handler/configurator/access_log.c.o
[ 36%] Building C object CMakeFiles/h2o.dir/lib/handler/configurator/expires.c.o
[ 38%] Building C object CMakeFiles/h2o.dir/lib/handler/configurator/file.c.o
[ 40%] Building C object CMakeFiles/h2o.dir/lib/handler/configurator/proxy.c.o
[ 42%] Building C object CMakeFiles/h2o.dir/lib/http1.c.o
[ 42%] Building C object CMakeFiles/h2o.dir/lib/http2/connection.c.o
[ 44%] Building C object CMakeFiles/h2o.dir/lib/http2/frame.c.o
[ 46%] Building C object CMakeFiles/h2o.dir/lib/http2/hpack.c.o
[ 48%] Building C object CMakeFiles/h2o.dir/lib/http2/scheduler.c.o
[ 48%] Building C object CMakeFiles/h2o.dir/lib/http2/stream.c.o
[ 51%] Building C object CMakeFiles/h2o.dir/src/main.c.o
/home/dch/src/www/h2o/src/main.c:235:11: warning: implicit declaration of function 'WIFEXITED' is invalid in C99 [-Wimplicit-function-declaration]
    if (!(WIFEXITED(child_status) && WEXITSTATUS(child_status) == 0))
          ^
/home/dch/src/www/h2o/src/main.c:235:38: warning: implicit declaration of function 'WEXITSTATUS' is invalid in C99 [-Wimplicit-function-declaration]
    if (!(WIFEXITED(child_status) && WEXITSTATUS(child_status) == 0))
                                     ^
/home/dch/src/www/h2o/src/main.c:238:86: warning: implicit declaration of function 'WTERMSIG' is invalid in C99 [-Wimplicit-function-declaration]
        fprintf(stderr, "[OCSP Stapling] command %s was killed by signal %d\n", cmd, WTERMSIG(child_status));
                                                                                     ^
3 warnings generated.
Linking C executable h2o
CMakeFiles/h2o.dir/src/main.c.o: In function `get_ocsp_response':
/home/dch/src/www/h2o/src/main.c:235: undefined reference to `WIFEXITED'
/home/dch/src/www/h2o/src/main.c:235: undefined reference to `WEXITSTATUS'
/home/dch/src/www/h2o/src/main.c:237: undefined reference to `WIFEXITED'
/home/dch/src/www/h2o/src/main.c:241: undefined reference to `WEXITSTATUS'
/home/dch/src/www/h2o/src/main.c:238: undefined reference to `WTERMSIG'
cc: error: linker command failed with exit code 1 (use -v to see invocation)
*** Error code 1

Stop.
make[2]: stopped in /usr/home/dch/src/www/h2o
*** Error code 1

Stop.
make[1]: stopped in /usr/home/dch/src/www/h2o
*** Error code 1

Stop.
make: stopped in /usr/home/dch/src/www/h2o
```

--- a/src/main.c
+++ b/src/main.c
@@ -39,6 +39,7 @@
 #include <sys/time.h>
 #include <sys/types.h>
 #include <sys/un.h>
+#include <sys/wait.h>
 #include <openssl/crypto.h>
 #include <openssl/err.h>
 #include <openssl/ssl.h>

